pandas/tests/frame/test_block_internals.py

```diff
@@ -585,10 +585,6 @@ def test_strange_column_corruption_issue(self):
         df = DataFrame(index=[0, 1])
         df[0] = np.nan
         wasCol = {}
-        # uncommenting these makes the results match
-        # for col in xrange(100, 200):
```

Comment:
This code block is sitting commented over for over 4 years. thtat's the only reason I deleted it all. and not just changed the ```xrange``` to ```range```